apps/backend/src/modules/activity-logs/activity-log.service.ts

```diff
@@ -87,6 +87,29 @@ class ActivityLogService implements Service {
 		}
 	}
 
+	private getAllowedProjectIds(
```

Comment:
we need to have this validation inside project service. We need to get user project permissions inside our common get all method for projects and then return only allowed projects. If we need to get a specific project this validation should still happen on project service level